internal/pkg/git/remote_test.go

```diff
@@ -23,6 +24,10 @@ import (
 
 func TestGit_Checkout(t *testing.T) {
 	t.Parallel()
+	if runtime.GOOS == "windows" {
+		t.Skip("unstable on windows - random timeouts")
+	}
+
```

Comment:
S tymito testami som to uz vzdal.

Nechcel som povodne ziadny test skipovat, tak sa tam pridalo retry, a teraz to pada nahodne na timeoutoch (niekedy to prejde niekedy nie). 

Asi by som to neriesil, kym s tym nebude mat problem nejaky realny uzivatel, alebo to nejak nasimulujeme (na MacOs a Linux sa to nedeje).



![image](https://user-images.githubusercontent.com/19371734/189917310-a0a5775d-eb0c-4fb3-aa84-c443402118a7.png)

Reply:
No já se spíš divím, že ti to vydrželo tak dlouho. 🙂